torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -852,6 +854,35 @@ def call_function(
         )
 
 
+class OutDtypeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+
```

Comment:
Is it possible to:
- check if there are any kwargs
- if so, then graph break (by calling `unimplemented`)?